### DISCUSSION TITLE
How Do I Fix "Cannot set properties of null" Error When Setting textContent in JavaScript?

### DISCUSSION THREAD

**Discussion by dewnaka7264** [community/community#136656] [Programming Help]
### Select Topic Area

Question

### Body

I'm working on a web project using JavaScript, and I keep getting the following runtime error in my console:

Uncaught TypeError: Cannot set properties of null (setting 'textContent')
    at HTMLDocument.<anonymous> (http://localhost:3000/:74:26)
I believe the error occurs when I'm trying to set the textContent of an HTML element. My code looks something like this:

document.getElementById('myElement').textContent = 'Hello, World!';
I've double-checked that the element ID is correct, and I’m sure that the script is loaded after the DOM elements. However, I’m still getting this error.

Does anyone know what might be causing this issue, or how I can fix it?

**Comment by ebndev** [comment]
Hi @dewnaka7264, thanks for being a part of the GitHub Community! 😄 

Can you share your code via code blocks or pushing it to a repo? The link is only accessible on your local machine so we can't see it. 

If anything you can test that you're targeting the element correctly first with

```
let myElement = document.getElementById('myElement')
console.log(myElement)
```

**Comment by KWIZERA-CALEB** [comment]
The link you provided is local and we can not visit. But i can try to help make sure your files are linked well and the element with id of **myElement** is in your markup . **Note**: Make sure the javascript file is linked after the all markup (html is loaded) move the link to the bottom of the html file 

```
<body>
  <div id="myElement"></div>
  <script>
    document.getElementById('myElement').textContent = 'Hello, World!';
  </script>
</body>
```
